app/models/miq_server/log_management.rb

```diff
@@ -124,13 +124,14 @@ def last_log_sync_message
   def post_logs(options)
     taskid = options[:taskid]
     task = MiqTask.find(taskid)
+    context = options[:context]
 
     # the current queue item and task must be errored out on exceptions so re-raise any caught errors
-    raise "Log depot settings not configured" unless log_depot
-    log_depot.update_attributes(:support_case => options[:support_case].presence)
+    raise "Log depot settings not configured" unless log_depot(context)
```

Comment:
let's store this depot as a variable, re-use it below, and pass it down to the other methods.